src/controllers/version/yank.rs

```diff
@@ -36,6 +38,15 @@ fn modify_yank(req: &mut dyn Request, yanked: bool) -> CargoResult<Response> {
         return Err(human("must already be an owner to yank or unyank"));
     }
 
+    if let AuthenticationSource::ApiToken(token_id) = req.authentication_source()? {
```

Comment:
What should I be doing in the else block? Getting the token in some other way? Returning an unauthorized error?